tests/queries/tests.py

```diff
@@ -432,6 +432,13 @@ def test_filter_by_related_field_nested_transforms(self):
                 Author.objects.filter(extra__info__ltrim__length=5), [a5]
             )
 
+    def test_filter_by_reverse_related_field_transform(self):
+        a5 = Author.objects.create(name="a5", extra=self.e2, num=5005)
+        r4 = Report.objects.create(name="r4", creator=a5)
+        fk_field = Report._meta.get_field("creator")
+        with register_lookup(fk_field, Abs):
+            self.assertSequenceEqual(Author.objects.filter(report__abs=r4.pk), [a5])
+
```

Comment:
It feels like the original tests and this one would have benefited from being categorized in the `lookups` test app instead of the mixed bag of tests that `queries` has become.

Reply:
Agreed, I move this test to `lookup`.